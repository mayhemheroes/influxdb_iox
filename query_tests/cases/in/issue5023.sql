-- IOX_SETUP: Issue5023

-- IOX_COMPARE: sorted
SELECT * FROM swap;
