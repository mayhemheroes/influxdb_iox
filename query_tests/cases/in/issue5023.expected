-- Test Setup: Issue5023
-- SQL: SELECT * FROM swap;
-- Results After Sorting
+------------+-----------+-----------+--------------+-----------+----------+----+-------+-----+----------------------+
| field_bool | field_f64 | field_i64 | field_string | field_u64 | host     | in | name  | out | time                 |
+------------+-----------+-----------+--------------+-----------+----------+----+-------+-----+----------------------+
|            |           |           |              |           | server01 | 3  | disk1 | 4   | 2020-06-11T16:54:20Z |
|            |           |           |              |           | server01 | 3  |       | 4   | 2020-06-11T16:54:10Z |
+------------+-----------+-----------+--------------+-----------+----------+----+-------+-----+----------------------+
